src/client/gameui.cpp

```diff
@@ -126,10 +126,10 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< "pos: (" << (player_position.X / BS)
 			<< ", " << (player_position.Y / BS)
 			<< ", " << (player_position.Z / BS)
-			<< "), yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
+			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "°"
```

Comment:
If you are not going to add a separator here, I suggest to keep at least the space as it is in master.

```suggestion
			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
```

Reply:
Will do, if reviewers don't see the need to separate yaw and cardinal direction.

Reply:
Restored the space. Thanks.